charts/radix-cicd-canary/templates/deployment.yaml

```diff
@@ -28,7 +28,7 @@ spec:
         - name: {{ .Chart.Name }}
           env:
             - name: LOG_LEVEL
-              value: DEBUG
+              value: {{ .Values.env.logLevel }}
```

Comment:
The LOG_LEVEL var should only be set if there is a value in .Values.env.logLevel

Wrap the LOG_LEVEL var in an if block, e.g.

```
{{- if .Values.env.logLevel }}
- name: LOG_LEVEL
  value: {{ .Values.env.logLevel }}
{{- end }}
```